packages/salesforcedx-vscode-core/resources/json_to_xsd_converter_clean.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
```

Comment:
is this just a local script that's run as part of creating the xsd?  If so, it shouldn't be in resources, it should be in a separate folder like `scripts` and `.vscodeignore` so we don't ship stuff in the vsix that users don't need.